app/views/schools/dashboard/show.html.erb

```diff
@@ -12,7 +12,7 @@
     <% @school_cohorts.each do |school_cohort| %>
 
       <% if school_cohort.core_induction_programme? || school_cohort.full_induction_programme? %>
-        <% if school_cohort.active_ecf_participant_profiles.any? %>
+        <% if school_cohort.active_ecf_participant_profiles.any? || school_cohort.induction_records.any? %>
```

Comment:
Should it be `active_induction_records` ?

Reply:
My thinking was that we are now showing all the various states including "not being trained" on the dashboard so I think if there are any induction records (in any state) we want to take them to the view?